pulp_ansible/tests/functional/utils.py

```diff
@@ -1,4 +1,5 @@
 """Utilities for tests for the ansible plugin."""
+from dynaconf import Dynaconf
```

Comment:
Our CI tests run client-side (outside the container), pointing to the server-side (container). 
`dynaconf` is installed server-side, so you need to add `dynaconf` here: https://github.com/pulp/pulp_ansible/blob/master/functest_requirements.txt to have it installed client-side

Reply:
What is the file "/etc/pulp/settings.py" for the CI? I need the directory "/tmp" to be in ALLOWED_EXPORT_PATHS and ALLOWED_IMPORT_PATHS

Reply:
it comes from template_config.yml:
https://github.com/pulp/pulp_file/blob/c0bc20b5e8d2aba354cb8c7a246816cd7c7c99ef/template_config.yml#L34
after adding it, you need to use plugin-template:
```
./plugin-template --github pulp_ansible
```
plugin-template converts settings from template_config.yml into CI scripts

Reply:
We set this here: https://github.com/pulp/pulp_ansible/blob/master/.ci/ansible/settings.py.j2#L9-L13

Comes from: https://github.com/pulp/pulpcore/blob/master/template_config.yml#L32-L42